diff --git a/index.d.ts b/index.d.ts
index a14cb68e6a7475248afa84b8e50915fadde08571..733864d16c05b2f6b5f816a6a20b096e196309d4 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,5 +1,7 @@
-// Despite not having any definitions, this file must exist on DefinitelyTyped.
-// Currently, the tooling assumes that this file exists and performs extra lints,
-// as well as using it to figure out where things are.
-
-export {};
+export {
+    default as createUploadLink,
+    ExtractableFileMatcher,
+    FormDataFileAppender,
+} from './createUploadLink.mjs';
+export { default as formDataAppendFile } from './formDataAppendFile.mjs';
+export { default as isExtractableFile, ExtractableFile } from './isExtractableFile.mjs';
diff --git a/package.json b/package.json
index 53648c43e3647251af6b47d9ca36b92ff911db1b..4fdbe3a97bf938481610babfc9731de9329507a5 100644
--- a/package.json
+++ b/package.json
@@ -17,8 +17,12 @@
         }
     ],
     "main": "",
-    "types": "index.d.ts",
     "exports": {
+        ".": {
+            "types": {
+                "default": "./index.d.ts"
+            }
+        },
         "./createUploadLink.mjs": {
             "types": "./createUploadLink.d.mts"
         },
